workflows/update_tags.sh

```diff
@@ -98,8 +98,22 @@ function update_tags_in_file() {
   sed -i -e "s|${THIS_IMAGE}:.*|${THIS_IMAGE}:${LATEST_TAG}|g" $THIS_FILE
 }
 
+function update_cray_sat_image() {
+  THIS_IMAGE=$1
+  # Capture the output of the podman search for img & update the img if it doesnt exist
+  SEARCH_OUTPUT=$(podman search ${DEFAULT_REGISTRY_NAME}/${THIS_IMAGE} 2>&1)
+  if [[ -z ${SEARCH_OUTPUT} ]]; then
+    THIS_IMAGE="artifactory.algol60.net/sat-docker/stable/cray-sat"
+  fi
+  echo "$THIS_IMAGE"
+}
+
 # Look up the latest tag for each image found and update the references in every file.
 for THIS_IMAGE in $(get_list_of_images_to_update); do
+  if [[ ${THIS_IMAGE} == *"cray-sat"* ]]; then
+    # CASMTRIAGE-7175 handle cray-sat image
+    THIS_IMAGE=$(update_cray_sat_image $THIS_IMAGE)
```

Comment:
Have you tested this on a system? You might consider running it on m003 in one of the lab environments.

Reply:
I had earlier tested the changes on starlord system were i had recreated manually the scenario. 
Now i have tested the unstable rpm on drax and verified it with proper system availability. Re-testing helped to catch hold of another bug in advance. Thank you @denniswalker !!